Lib/unittest/async_case.py

```diff
@@ -104,7 +104,7 @@ async def _asyncioLoopRunner(self, fut):
                     fut.set_result(ret)
             except asyncio.CancelledError:
```

Comment:
Please add `except (SystemExit, KeyboardInterrupt): raise` before handling the `BaseException`.

Reply:
@1st1 this will still leave `SystemExit` and `KeyboardInterrupt` hanging if they are raised, I think they should be handled along with the `BaseException`.